### PR TITLE
OOUI

### PR DESCRIPTION
**Pull request by elifoster**
Update our special pages to use OOUI, using HTMLForm where possible. Currently, the only places where it is not possible to use HTMLForm is the search fields.

### CODE REVIEW

**Comment by elifoster** [comment]
@Alexia Would you mind doing a code review on this?

**Comment by elifoster** [comment]
Also, @xbony2 @retep998 please review the new look:

**Special:CreateTileSheet** (note this is zoomed out to fit it all in the window):
Screen Shot 2019-12-17 at 2 12 26 PM

**Special:SheetList**:
Screen Shot 2019-12-17 at 2 13 48 PM

**Special:SheetManager**:
Screen Shot 2019-12-17 at 2 14 39 PM

**Special:TileList**:
Screen Shot 2019-12-17 at 2 15 14 PM

**Special:TileManager**:
Screen Shot 2019-12-17 at 2 15 48 PM

**Special:TileTranslator**:
Screen Shot 2019-12-17 at 2 16 07 PM

File: special/CreateTileSheet.php [hunk 1]
```diff
@@ -88,37 +89,58 @@ public function execute($par) {
...
-		$out = Xml::openElement('form', array('method' => 'post', 'action' => str_replace('$1', 'Special:CreateTileSheet', $wgArticlePath), 'id' => 'ext-tilesheet-create-form', 'class' => 'prefsection')) .
-			Xml::fieldset($this->msg('tilesheet-create-legend')->text()) .
-			Html::hidden('title', $this->getTitle()->getPrefixedText()) .
-			Html::hidden( 'token', $this->getUser()->getEditToken() ) .
-			$form .
-			Xml::closeElement( 'fieldset' ) . Xml::closeElement( 'form' ) . "\n";
-
-		return $out;
+	private function displayForm() {
+	    global $wgArticlePath;
```
Comment (Alexia) [inline special/CreateTileSheet.php]:
```suggestion
        $mainConfig = MediaWikiServices::getInstance()->getMainConfig();
        $wgArticlePath->get('ArticlePath');
```

If you would like to get fancy with future compatibility this eliminates the need for using the `global`.

**Comment by Alexia** [comment]
I am not too familiar with OOUI, but I do not see anything that is out of place.

**Comment by xbony2** [comment]
seems fine to me

File: special/CreateTileSheet.php (hunk 1 above)
Comment (elifoster) [reply special/CreateTileSheet.php]: `global` is used throughout the extension (beyond UI stuff) so I'll go through and update them all after this PR, and do the same on OreDict.

**Pull request merged**